File: 4/simple_database.cs
using MySql.Data.MySqlClient;
using System;

class Program
{
    static void Main(string[] args)
    {
        string connStr = "server=localhost;user=root;password=P@ssw0rd12!;database=test";
        MySqlConnection conn = new MySqlConnection(connStr);
        try
        {
            Console.WriteLine("Connecting to MySQL...");
            conn.Open();

            string sql = "SHOW DATABASES";
            MySqlCommand cmd = new MySqlCommand(sql, conn);
            MySqlDataReader rdr = cmd.ExecuteReader();

            while (rdr.Read())
            {
                Console.WriteLine(rdr[0]);
            }
            rdr.Close();
        }
        catch (Exception ex)
        {
            Console.WriteLine(ex.ToString());
        }
        conn.Close();
        Console.WriteLine("Done.");
    }
}

File: 4/simple_database.py
import mysql.connector

# Connect to MySQL connector.
mydb = mysql.connector.connect(
    host="localhost",
    user="root",
    password="P@ssw0rd12!")

mycursor = mydb.cursor()


# Create database for products.
mycursor.execute("SHOW DATABASES")
for db in mycursor:
    print(db)
